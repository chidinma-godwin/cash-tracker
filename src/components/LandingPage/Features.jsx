// CashTracker imports
import useStyles from './useStyles';

export default function Features() {
  const styles = useStyles();

  return (
    <section className={styles.subSection}>
      <h2>FEATURES</h2>
      <p>
        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ducimus ipsum
        quas eum saepe molestiae numquam dolores autem facilis! Veniam harum nam
        dolore cumque, sed illum eligendi incidunt unde earum modi.
      </p>
      <p>
        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Molestiae
        ullam quia animi voluptatem autem, temporibus dolore, pariatur ipsam
        reprehenderit sapiente numquam nobis molestias asperiores magnam culpa
        ratione deserunt id dignissimos.
      </p>
    </section>
  );
}
