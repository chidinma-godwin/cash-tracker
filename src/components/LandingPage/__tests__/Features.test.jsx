import { render } from '@testing-library/react';
import Features from '../Features';

describe('Features', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Features />);
    expect(asFragment(<Features />)).toMatchSnapshot();
  });
});
