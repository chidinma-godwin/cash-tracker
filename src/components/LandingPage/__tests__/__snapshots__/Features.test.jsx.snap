// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="makeStyles-subSection-4"
  >
    <h2>
      FEATURES
    </h2>
    <p>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ducimus ipsum quas eum saepe molestiae numquam dolores autem facilis! Veniam harum nam dolore cumque, sed illum eligendi incidunt unde earum modi.
    </p>
    <p>
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Molestiae ullam quia animi voluptatem autem, temporibus dolore, pariatur ipsam reprehenderit sapiente numquam nobis molestias asperiores magnam culpa ratione deserunt id dignissimos.
    </p>
  </section>
</DocumentFragment>
`;
